From 45be172a9b5ce6fcf8bab89685ea88955192d474 Mon Sep 17 00:00:00 2001
From: Dooley_labs <elderlabs@users.noreply.github.com>
Date: Mon, 15 Jul 2019 04:57:57 -0400
Subject: [PATCH] Patch Python 3.8 deprecation warning

This warning has been plaguing me. This fixes it. Forever.
```
py.warnings:110 - .../pygal/_compat.py:23: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
```
---
 pygal/_compat.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/pygal/_compat.py b/pygal/_compat.py
index 07ab8846..2c6ab12b 100644
--- a/pygal/_compat.py
+++ b/pygal/_compat.py
@@ -20,8 +20,11 @@
 from __future__ import division
 
 import sys
-from collections import Iterable
 from datetime import datetime, timedelta, tzinfo
+try:
+    from collections.abc import Iterable
+except ImportError:
+    from collections import Iterable
 
 if sys.version_info[0] == 3:
     base = (str, bytes)
